doorstop/gui/utilTkinter.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+import sys
+from unittest.mock import Mock
+try:  # pragma: no cover (manual test)
+    import tkinter as tk
+    from tkinter import ttk
+except ImportError as _exc:  # pragma: no cover (manual test)
+    sys.stderr.write("WARNING: {}\n".format(_exc))
+    tk = Mock()
+    ttk = Mock()
```

Comment:
We may not need this pattern anymore. I think I added it for testing, but I'm not sure how much value we can get out of unit testing this GUI.

Reply:
Ok, I simply copy pasted the import logic from the other file.  We can keep it like that for now, at least the import strategy is the same everywhere.